Lib/test/test_site.py

```diff
@@ -805,5 +806,112 @@ def test_underpth_dll_file(self):
         self.assertTrue(rc, "sys.path is incorrect")
 
 
+class CommandLineTests(unittest.TestCase):
+    def exists(self, path):
+            if path is not None and os.path.isdir(path):
+                return "exists"
+            else:
+                return "doesn't exist"
```

Comment:
This is indented too deeply:
```suggestion
        if path is not None and os.path.isdir(path):
            return "exists"
        else:
            return "doesn't exist"
```